week04_approx_rl/homework_tf.ipynb

```diff
@@ -87,7 +147,7 @@
     "from gym.core import ObservationWrapper\n",
     "from gym.spaces import Box\n",
     "\n",
-    "import cv2\n",
+    "import skimage\n",
     "\n",
     "\n",
     "class PreprocessAtari(ObservationWrapper):\n",
```

Comment:
This wrapper is called `PreprocessAtari`, yet down below you refer to it as `PreprocessAtariObs`.